component/file_cache/file_cache_linux.go

```diff
@@ -75,6 +77,25 @@ func (fc *FileCache) isDownloadRequired(localPath string, blobPath string, flock
 		log.Debug("FileCache::isDownloadRequired : %s not present in local cache policy", localPath)
 		downloadRequired = true
 	}
+	//check async map and automatically set downloadRequired false if entry is matched
+	//When cloud is down, local data will always be most up to date compared to cloud, download is not required
+	//Async map stores file entries that have been changed when cloud is down
+
+	//case is no record of file in cloud
+	//does not exist locally
+	//this does not matter because isDownloadRequired is called by OpenFile, which means file already exists somewhere
+	fc.fileOps.Range(func(key, value interface{}) bool {
```

Comment:
Could you cleanup some of the empty lines here.

Reply:
Done